.storybook/addons.js

```diff
@@ -1,3 +1,2 @@
 import '@storybook/addon-notes/register';
 import '@storybook/addon-knobs/register';
-import 'storybook-addon-rtl/register';
```

Comment:
We shouldn't remove these until we use this package elsewhere :)

Reply:
Does it also mean that we shouldn't change the config?

Reply:
We shouldnt remove `storybook-addon-rtl` untill we use the new dirDecorator everywhere